packages/paste-website/src/pages/core/changing-theme.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Changing the Theme
+description: When using Paste, migrating to a different theme is a simple two step process.
+slug: /core/changing-theme
+---
+
+import {graphql} from 'gatsby';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/core/changing-theme/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.CORE}
+/>
+
+---
+
+ <contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Update the theme provider
+
+The `<Theme.Provider>` is a wrapper component that provides a theme object to any descendant. It is usually towards the root of your app.
```

Comment:
Copy update suggestion:

> "It is usually towards the root of your app."

"It is usually placed at or near the root of your app."